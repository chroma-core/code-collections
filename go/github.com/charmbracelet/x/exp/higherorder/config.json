{
  "native_identifier": "github.com/charmbracelet/x/exp/higherorder",
  "repo": "charmbracelet/x",
  "registry": "go",
  "tag_formats": [
    "/v{major}.{minor}.{patch}",
    "ansi/v{major}.{minor}.{patch}",
    "ansi/v{major}.{patch}.{minor}"
  ],
  "sentinel_timestamp": "2024-01-01T00:00:00+0000",
  "include": [
    "exp/higherorder/**/*.md",
    "exp/higherorder/**/*.go"
  ]
}