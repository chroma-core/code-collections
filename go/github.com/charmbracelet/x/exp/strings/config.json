{
	"native_identifier": "github.com/charmbracelet/x/exp/strings",
	"repo": "charmbracelet/x",
	"registry": "go",
	"tag_formats": [
		"/v{major}.{minor}.{patch}",
		"ansi/v{major}.{minor}.{patch}",
		"ansi/v{major}.{minor}.0",
		"ansi/v{major}.{patch}.0"
	],
	"sentinel_timestamp": "2020-01-01T00:00:00Z",
	"include": [
		"exp/strings/**/*.md",
		"exp/strings/**/*.go"
	],
	"collection_name_prefix": "github.com_charmbracelet_x_exp_strings"
}
