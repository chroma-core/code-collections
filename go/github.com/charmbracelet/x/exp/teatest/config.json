{
  "native_identifier": "github.com/charmbracelet/x/exp/teatest",
  "repo": "charmbracelet/x",
  "registry": "go",
  "tag_formats": [
    "/v{major}.{minor}.{patch}",
    "ansi/v{major}.{minor}.{patch}"
  ],
  "sentinel_timestamp": "2024-01-01T00:00:00+0000",
  "include": [
    "exp/teatest/**/*.md",
    "exp/teatest/**/*.go"
  ]
}