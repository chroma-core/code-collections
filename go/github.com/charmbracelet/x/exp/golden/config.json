{
  "native_identifier": "github.com/charmbracelet/x/exp/golden",
  "repo": "charmbracelet/x",
  "registry": "go",
  "tag_formats": [
    "/v{major}.{minor}.{patch}",
    "ansi/v{major}.{minor}.{patch}",
    "ansi/v{major}.{patch}.{minor}"
  ],
  "sentinel_timestamp": "2024-01-01T00:00:00+0000",
  "include": [
    "exp/golden/**/*.md",
    "exp/golden/**/*.go"
  ]
}